moj.json

```diff
@@ -0,0 +1,33 @@
+{
+  "repositories": [
+    {"type":"composer", "url":"https://wpackagist.org"},
+    {"type":"composer", "url":"https://composer.wp.dsd.io"},
+    {"type":"vcs", "no-api": true, "url":"https://github.com/ministryofjustice/dw-mvc.git"},
+    {"type":"vcs", "no-api": true, "url":"https://github.com/ministryofjustice/dw-document-revisions.git"},
+    {"type":"vcs", "no-api": true, "url":"https://github.com/ministryofjustice/dw-live-drafts.git"},
+    {"type":"vcs", "no-api": true, "url":"https://github.com/ministryofjustice/dw-markdown.git"},
+    {"type":"vcs", "no-api": true, "url":"https://github.com/ministryofjustice/intranet-theme-clarity.git"},
+    {"type":"vcs", "no-api": true, "url":"https://github.com/ministryofjustice/intranet.git"},
+    {"type":"vcs", "no-api": true, "url":"https://github.com/ministryofjustice/like-button-for-wordpress.git"}
+  ],
+  "require": {
+    "wpackagist-plugin/breadcrumb-navxt":"5.7.1",
+    "wpackagist-plugin/cms-tree-page-view":"1.3.4",
+    "wpackagist-plugin/co-authors-plus":"3.2.2",
+    "wpackagist-plugin/recently-edited-content-widget":"0.3.2",
+    "wpackagist-plugin/regenerate-thumbnails":"2.2.6",
+    "wpackagist-plugin/upload-url-path-enabler":"1.0.4",
+    "relevanssi/relevanssi-premium":"2.0.3.1",
+    "acf/advanced-custom-fields-pro":"5.4.4",
+
+    "ministryofjustice/dw-mvc":"dev-master",
+    "ministryofjustice/dw-document-revisions":"dev-update-composer-json",
+    "ministryofjustice/dw-live-drafts":"dev-update-composer-json",
+    "ministryofjustice/dw-markdown":"dev-update-composer-json",
+    "ministryofjustice/intranet-theme-clarity":"dev-master",
+    "ministryofjustice/intranet":"dev-master",
+    "ministryofjustice/wp-rewrite-media-to-s3": "*",
+    "ministryofjustice/like-button-for-wordpress":"dev-master",
+    "wpackagist-theme/twentyseventeen":"1.3"
```

Comment:
"wpackagist-theme/twentyseventeen":"1.3" This line can go, not using this theme whatsoever.

Reply:
I wondered...It's been in there forever, but didn't seem to be doing anything.